tutoraspects/patches/openedx-common-settings

```diff
@@ -7,10 +7,22 @@ EVENT_SINK_CLICKHOUSE_BACKEND_CONFIG = {
 	"timeout_secs": {{ ASPECTS_EVENT_SINK_CLICKHOUSE_TIMEOUT_SECS }}
 }
 SUPERSET_CONFIG = {
-	"service_url": "http://superset:{{ SUPERSET_PORT }}/",
-	"host": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ SUPERSET_HOST }}",
+	"internal_service_url": "http://superset:{{ SUPERSET_PORT }}/",
+	"service_url": "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ SUPERSET_HOST }}",
 	"username": "{{ SUPERSET_LMS_USERNAME }}",
 	"password": "{{ SUPERSET_LMS_PASSWORD }}",
-	"email": "{{ SUPERSET_LMS_EMAIL }}",
 }
 EVENT_SINK_CLICKHOUSE_PII_MODELS = {{ EVENT_SINK_PII_MODELS }}
+
+ASPECTS_INSTRUCTOR_DASHBOARD_UUID = "{{ ASPECTS_INSTRUCTOR_DASHBOARD_UUID }}"
+SUPERSET_EXTRA_FILTERS_FORMAT = {{ ASPECTS_SUPERSET_EXTRA_FILTERS_FORMAT }}
+{% if ASPECTS_ENABLE_INSTRUCTOR_DASHBOARD_PLUGIN %}
+OPEN_EDX_FILTERS_CONFIG = {
+   "org.openedx.learning.instructor.dashboard.render.started.v1": {
+     "fail_silently": False,
+     "pipeline": [
+       "platform_plugin_aspects.extensions.filters.AddSupersetTab",
+     ]
+   },
+}
+{% endif %}
```

Comment:
Is there a way to do this without overwriting an existing `OPEN_EDX_FILTERS_CONFIG`?

Reply:
The config change ended up being a little more complicated than this because `OPEN_EDX_FILTERS_CONFIG` isn't defined yet: [1d8bec1](https://github.com/openedx/tutor-contrib-aspects/pull/642/commits/1d8bec16f9c8a6ed2cfc5edb83fd2479bb641720)

I expect other filters could use logic like this too, so we may want to move it somewhere more general eventually.